lib/karaokium_web/live/admin/performance_live/show.ex

```diff
@@ -15,9 +16,30 @@ defmodule KaraokiumWeb.Admin.PerformanceLive.Show do
      socket
      |> assign(:page_title, page_title(socket.assigns.live_action))
      |> assign(:karaoke_id, karaoke_id)
-     |> assign(:performance, Performances.get_performance!(id))}
+     |> assign(:id, id)
+     |> assign(:performancelive, Events.get_karaoke!(karaoke_id).performing_id)
```

Comment:
```suggestion
     |> assign(:performance_id, Events.get_karaoke!(karaoke_id).performing_id)
```

What do you think about changing the name of this assign to make it more explicit on what it is?

Reply:
Ok. In my next commit for another feature, if I don't forget, I will change it.